src/zc.adoc

```diff
@@ -0,0 +1,2611 @@
+----
+
+<<<
+[#insns-c_sext_h,reftext="Sign extend halfword, 16-bit encoding"]
+==== c.sext.h
+
+Synopsis:
+
+Sign extend halfword, 16-bit encoding
+
+Mnemonic:
+
+c.sext.h _rd'/rs1'_
+
+Encoding (RV32, RV64):
+
+[wavedrom, , svg]
+....
+{reg:[
+    { bits:  2, name: 0x1, attr: ['C1'] },
+    { bits:  3, name: 0x3, attr: ['C.SEXT.H'] },
+    { bits:  2, name: 0x3, attr: ['FUNCT2'] },
+    { bits:  3, name: 'rd\'/rs1\'', attr: ['SRCDST'] },
+    { bits:  3, name: 0x7 },
+    { bits:  3, name: 0x4, attr: ['FUNCT3'] },
+],config:{bits:16}}
+....
+
+Description:
+
+This instruction takes a single source/destination operand. 
+It sign-extends the least-significant halfword in the operand to XLEN bits by copying the most-significant bit
+in the halfword (i.e., bit 15) to all of the more-significant bits.
+
+[NOTE]
+====
+_rd'/rs1'_ is from the standard 8-register set x8-x15.
+====
+
+Prerequisites:
+
+Zbb is also required.
+//
+//32-bit equivalent:
+//
+//<<insns-sext_h>> from Zbb
+
+[NOTE]
+====
+The SAIL module variable for _rd'/rs1'_ is called _rsdc_.
+====
+
+Operation:
+
+[source,sail]
+----
+X(rsdc) = EXTS(X(rsdc)[15..0]);
+----
+
+<<<
+[#insns-c_zext_w,reftext="Zero extend word, 16-bit encoding"]
+==== c.zext.w
+
+Synopsis:
+
+Zero extend word, 16-bit encoding
+
+Mnemonic:
+
+c.zext.w _rd'/rs1'_
+
+Encoding (RV64):
+
+[wavedrom, , svg]
+....
+{reg:[
+    { bits:  2, name: 0x1, attr: ['C1'] },
+    { bits:  3, name: 0x4, attr: ['C.ZEXT.W'] },
+    { bits:  2, name: 0x3, attr: ['FUNCT2'] },
+    { bits:  3, name: 'rd\'/rs1\'', attr: ['SRCDST'] },
+    { bits:  3, name: 0x7 },
+    { bits:  3, name: 0x4, attr: ['FUNCT3'] },
+],config:{bits:16}}
+....
+
+Description:
+
+This instruction takes a single source/destination operand. 
+It zero-extends the least-significant word of the operand to XLEN bits by inserting zeros into all of
+the bits more significant than 31.
+
+[NOTE]
+====
+_rd'/rs1'_ is from the standard 8-register set x8-x15.
+====
+
+Prerequisites:
+
+Zba is also required.
+ 
+32-bit equivalent:
+
+[source,sail]
+----
+add.uw rd'/rs1', rd'/rs1', zero
+----
+
+[NOTE]
+====
+The SAIL module variable for _rd'/rs1'_ is called _rsdc_.
+====
+
+Operation:
+
+[source,sail]
+----
+X(rsdc) = EXTZ(X(rsdc)[31..0]);
+----
+
+<<<
+[#insns-c_not,reftext="Bitwise not, 16-bit encoding"]
+==== c.not
+
+Synopsis:
+
+Bitwise not, 16-bit encoding
+
+Mnemonic:
+
+c.not _rd'/rs1'_
+
+Encoding (RV32, RV64):
+
+[wavedrom, , svg]
+....
+{reg:[
+    { bits:  2, name: 0x1, attr: ['C1'] },
+    { bits:  3, name: 0x5, attr: ['C.NOT'] },
+    { bits:  2, name: 0x3, attr: ['FUNCT2'] },
+    { bits:  3, name: 'rd\'/rs1\'', attr: ['SRCDST'] },
+    { bits:  3, name: 0x7 },
+    { bits:  3, name: 0x4, attr: ['FUNCT3'] },
+],config:{bits:16}}
+....
+
+Description:
+
+This instruction takes the one's complement of _rd'/rs1'_ and writes the result to the same register.
+
+[NOTE]
+====
+  _rd'/rs1'_ is from the standard 8-register set x8-x15.
```

Comment:
the _'s haven't caused italics

Reply:
That's due to the theme italicizing admonitions by default.  This is consistent with the rest of the spec.